tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -68,6 +68,141 @@ public async Task CanBuildAndRun_Angular6NodeApp_WithoutZippingNodeModules(strin
                 });
         }
 
+        [Theory]
+        [InlineData("8")]
+        [InlineData("9.4")]
+        public async Task CanBuildAndRun_Angular6NodeApp_With_Node_Module_Dir_Exists_InRoot__WithoutZipping(string nodeVersion)
+        {
+            // Arrange
+            var appName = "angular6app";
+            var volume = CreateAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var buildScript = new ShellScriptBuilder()
+               .AddCommand($"oryx build {appDir} --platform nodejs --platform-version {nodeVersion}")
+               .ToString();
+            var runScript = new ShellScriptBuilder()
+                .SetEnvironmentVariable("PORT", "4200")
+                .AddCommand("mkdir -p node_modules")
```

Comment:
where is this folder getting created? It should be created under `appDir`

Reply:
this is for  root, have additional tests covering this